core/src/main/scala/org/apache/spark/deploy/mesos/MesosClusterDispatcherArguments.scala

```diff
@@ -95,6 +100,9 @@ private[mesos] class MesosClusterDispatcherArguments(args: Array[String], conf:
         "  -h HOST, --host HOST    Hostname to listen on\n" +
         "  -p PORT, --port PORT    Port to listen on (default: 7077)\n" +
         "  --webui-port WEBUI_PORT WebUI Port to listen on (default: 8081)\n" +
+        "  --webui-url  WEBUI_URL  The WebUI URL to be set in the underlying\n" +
```

Comment:
We could only take `spark.mesos.dispatcher.webui.url`, sure. I thought this is idiomatic in this context.